src/App.jsx

```diff
@@ -0,0 +1,19 @@
+import { Route, Routes } from "react-router";
+import AddItem from "./components/AddItem/AddItem";
+import Nav from "./common/Nav/Nav";
+import Items from "./components/Items/Items";
+import Products from "./components/Products/Products";
+
+export default function App() {
+  return (
+    <>
+      <Nav />
+      <Routes>
+        <Route path="/" element={<AddItem />} />
```

Comment:
💬 여담
index 속성을 이용해 시작페이지라는 것을 더 명확하게 알 수 있게 하실수도 있습니다~
```suggestion
        <Route index element={<AddItem />} />
```